src/app/donation-start/donation-start-form/donation-start-form-new.component.html

```diff
@@ -320,7 +321,7 @@ <h3 class="b-rt-0 b-m-0 b-bold"><span class="span-hr"></span>How does Big Give u
 
           <div>
             <biggive-text-input  prompt="Email Address *">
-              <input slot="input" formControlName="emailAddress" id="emailAddress" type="email" matInput>
+              <input slot="input" formControlName="emailAddress" id="emailAddress" type="email" autocomplete="email" matInput>
```

Comment:
Having a `label` element associated with the input might be a better or good additional way to make the autocomplete work - but I can't see an easy way to do that given that the input element is here and if we had a label it would need to be positioned as part of the components in the components library.

Reply:
There might be a more important point here about accessibility @bdsl – have we tried the new approach with a screen reader?

If `prompt` is not used in an ARIA attribute or an actual `label`, I'm not sure we are really associating it semantically in a way that will keep the form usable for everyone.

Reply:
@NoelLH You're right, we should start using labels. I've made a ticket for that https://thebiggive.atlassian.net/browse/DON-850